### DISCUSSION TITLE
rspack & rsbuild

### DISCUSSION THREAD

**Discussion by zknic** [quasarframework/quasar#17076] [Ideas / Proposals]
Quasar need rsbuild!

https://www.rspack.dev/guide/introduction.html

![image](https://github.com/quasarframework/quasar/assets/11263870/e44b79d8-6da2-467b-895b-71f2eab3129e)
![image](https://github.com/quasarframework/quasar/assets/11263870/84cf62fe-a233-41bd-8f79-7cab1bbd2e10)

**Comment by gowy222** [comment]
+1
https://github.com/web-infra-dev/rspack

**Comment by stefanvanherwijnen** [comment]
In the future Vite will switch to [rolldown](https://rolldown.rs/), which should result in similar performance increases.

**Reply by tmcdos** [reply]
Vite will be soon replaced by [VoidZero](https://voidzero.dev/)

**Reply by stefanvanherwijnen** [reply]
No it won't.
https://voidzero.dev/posts/announcing-voidzero-inc

Vite will use the VoidZero toolchain in the future (e.g. rolldown).
